src/routes/api/url.ts

```diff
@@ -0,0 +1,32 @@
+import type { RequestHandler } from '@sveltejs/kit';
+import { Prisma } from '../../database/prisma';
+
+export const get: RequestHandler = async ({ request }) => {
+	const url = new URL(request.url);
+	const query = url.searchParams.get('search')?.trim() ?? undefined;
+
+	if (query === undefined || query === null) {
+		return {
+			status: 200,
+			body: true
+		};
+	}
+
+	const collab = await Prisma.client.collab.findMany({
+		where: {
+			url: query
+		}
+	});
+
+	if (collab.length === 0) {
+		return {
+			status: 200,
+			body: true
+		};
+	} else {
+		return {
+			status: 200,
+			body: false
+		};
+	}
+};
```

Comment:
Not necessary, add a unique constraint to url if it's not there yet, then query in the api collab id endpoint with an or on url and id